plugin/storage/badger/config.go

```diff
@@ -79,9 +73,9 @@ const (
 	defaultKeysDir            = defaultDataDir + string(os.PathSeparator) + "keys"
 )
 
-func DefaultNamespaceConfig() NamespaceConfig {
+func NewConfig() *Config {
```

Comment:
"new config" is not the same as "default config".

```suggestion
func DefaultConfig() *Config {
```